### PR TITLE
[LA-15858] Add information about optional Timezone param

### PR DESCRIPTION
**Pull request by pajw**
[LA-15858](https://learnamp.atlassian.net/browse/LA-15858)

Add information about optional `:timezone` param to Activities API

image

[LA-15858]: https://learnamp.atlassian.net/browse/LA-15858?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: source/includes/_activities.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ expanded | true | Optional expanded json. Returns certificate and any related ex
 include_deactivated_users | true | Optional - default is false. When true activity returned by the end-point will include activity by deactivated users.
 filters[date][from] | "2021-12-31" | Date range FROM date in ISO 8601 format
 filters[date][to] | "2022-02-28" | Date range TO date in ISO 8601 format
+filters[timezone] | "UTC" | Specify Timezone by which to filter activites. See [Rails Timezone names](https://apidock.com/rails/ActiveSupport/TimeZone).
```
Comment (ttrmw) [inline source/includes/_activities.md:75]: I would replicate the list here rather

**Review by ttrmw — CHANGES_REQUESTED** [review]
.

File: source/includes/_activities.md (hunk 1 above)
Comment (pajw) [reply source/includes/_activities.md:75]: and reference this page on the API error response yeah?

Comment (pajw) [reply source/includes/_activities.md:75]: this is live btw so can you help me fix it

**Review by ttrmw — APPROVED** [review]

**Pull request merged**